dataland-frontend/src/components/general/TheJoinCampaign.vue

```diff
@@ -0,0 +1,218 @@
+<template>
+  <section v-if="joinCampaignSection" class="joincampaign" role="region" aria-labelledby="joincampaign-heading">
+    <div class="joincampaign__wrapper">
+      <h2 id="joincampaign-heading" class="joincampaign__title">
+        {{ joinCampaignSection.text[0] }}
+        <span v-if="joinCampaignSection.text[1]"> {{ joinCampaignSection.text[1] }}</span
+        >{{ joinCampaignSection.text[2] }}
+      </h2>
+      <p class="joincampaign__paragraph" v-if="joinCampaignSection.text[3]">{{ joinCampaignSection.text[3] }}</p>
+      <email-button :sections="sections" />
```

Comment:
Can we write "EmailButton" here instead of "email-button"?
Reason: My IDE couldn't find this usage of that component, because it is spelled differently than the original component (EmailButton.vue).

Reply:
Yes we can, I changed it.